src/commands.js

```diff
@@ -409,3 +409,26 @@ export function deleteTable(state, dispatch) {
   }
   return false
 }
+
+// :: (?number, ?number, ?Object) → (EditorState, dispatch: ?(tr: Transaction)) → bool
+// Creates a table with specified size and attrs, then focus to first cell.
+export function createTable(row = 1, col = 1, cellAttrs = {}) {
+  return function(state, dispatch) {
+    const { tr, schema } = state
+    const tableType = schema.nodes.table
+    const rowType = schema.nodes.table_row
```

Comment:
I believe these should not be hard-coded, instead you should use the type look-up function.

Reply:
`rowType` is mutable indeed, while how about `tableType`?

Reply:
Check out `tableNodeTypes(schema)` in `src/schema.js`. It will provide you with the node types keyed by table role. It searches through all the node types in the schema and checks their `tableRole`, then caches and returns the result.

This feature allows the node type names to be provided by the user.